Final changes responding to JOSS comments (#76)
aiidateam/kiwipy: docs/source/installation.rst

Installation
============

Python
------

> KiwiPy supports Python versions 3.5 and above, however the minimum supported patch versions are 3.5.4 and 3.6.2 for Python 3.6.

Reviewer: ```suggestion
KiwiPy supports Python versions 3.5 and above, however, the minimum required patch versions are 3.5.4 and 3.6.2 for Python 3.5 and 3.6, respectively.
```

All newer versions are supported.

RabbitMQ
--------

> kiwipy depends on RabbitMQ as the message broker.  On ubuntu this is as simple as:

Reviewer: ```suggestion
KiwiPy depends on RabbitMQ as the message broker. 
On ubuntu this is as simple as:
```

    . venv/bin/activate
    pip install -e "kiwipy[rmq,pre-commit,tests]"

> To run the tests, make sure you're running the RabbitMQ server (see the RabbitMQ documentation on how to do this) and type:

Reviewer: ```suggestion
To run the tests, make sure the RabbitMQ server is up and running (see the RabbitMQ documentation on how to accomplish and/or verify this) and type:
```
